gradle-plugins/preview-rpc/build.gradle.kts

```diff
@@ -0,0 +1,32 @@
+plugins {
```

Comment:
Why we need separate preview-rpc Gralde plugin?

Reply:
It's not a plugin, it's a separate module, that is used by both the Gradle plugin and the Intellij plugin. It needs to be separate module, because I don't want to introduce a compile/runtime dependency between the Gradle plugin and the Intellij plugin.